extension/src/App.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  createSignal,
+  createEffect,
+  createRoot,
+  getOwner,
+  Switch,
+  Match,
+} from 'solid-js';
+import { Header } from './components/Header';
+import { Inspect } from './components/Inspect';
+import { Graph } from './components/Graph';
+import { Navbar } from './components/Navbar';
+import { SolidComponent, TabType, OrientType } from './types';
+import './styles/main.scss';
+
+export const SolidStructure: SolidComponent = () => {
+  // const [children, root] = createRoot(() => [props.children, getOwner()!]);
```

Comment:
Are we accessing browser-rendered signals through createRoot()?

Reply:
The idea is that we are wrapping the target application in our SolidStructure component and with the addition of a createRoot, we will have access to all Signals and Signals of children components.